pom.xml

```diff
@@ -74,7 +74,7 @@
 			<dependency>
 				<groupId>com.github.mpkorstanje</groupId>
 				<artifactId>simmetrics-core</artifactId>
-				<version>5.0.0-SNAPSHOT</version>
+				<version>4.1.1-klara.1-SNAPSHOT</version>
```

Comment:
Why do we need to repeat the version. Can't we use the maven property here?

Reply:
Which maven property?